src/app/leaves/actions/process-leave-request.test.ts

```diff
@@ -0,0 +1,19 @@
+import { waitFor } from "@testing-library/dom";
+import { processLeaveRequest } from "./process-leave-request";
+
+describe("delete leave request", () => {
+  const mockPlayload = {
+    requestId: 1,
+    requestStatus: "rejected",
```

Comment:
@nadirhussain786  use Enum for `rejected`

Reply:
Fixed